client/components/TestManagement/StatusSummaryRow/index.jsx

```diff
@@ -0,0 +1,200 @@
+import { Dropdown } from 'react-bootstrap';
+import nextId from 'react-id-generator';
+import { BULK_UPDATE_TEST_PLAN_REPORT_STATUS_MUTATION } from '../queries';
+import { LoadingStatus, useTriggerLoad } from '../../common/LoadingStatus';
+import BasicThemedModal from '../../common/BasicThemedModal';
+
+const PhaseText = styled.span`
+    font-size: 14px;
+    margin-left: 8px;
+    padding: 4px 8px;
+    border-radius: 14px;
+    overflow: hidden;
+    white-space: nowrap;
+    color: white;
+
+    &.draft {
+        background: #838f97;
+    }
+
+    &.candidate {
+        background: #f87f1b;
+    }
+
+    &.recommended {
+        background: #b253f8;
+    }
+`;
+
+const PhaseDot = styled.span`
+    display: inline-block;
+    height: 10px;
+    width: 10px;
+    padding: 0;
+    margin-right: 8px;
+    border-radius: 50%;
+
+    &.draft {
+        background: #838f97;
+    }
+
+    &.candidate {
+        background: #f87f1b;
+    }
+
+    &.recommended {
+        background: #b253f8;
+    }
+`;
+
+const StatusSummaryRow = ({ atItems }) => {
+    const [bulkUpdateTestPlanReportStatusMutation] = useMutation(
+        BULK_UPDATE_TEST_PLAN_REPORT_STATUS_MUTATION
+    );
+
+    const dropdownUpdateReportStatusButtonRef = useRef();
+    const { triggerLoad, loadingMessage } = useTriggerLoad();
+
+    const [testPlanReports, setTestPlanReports] = useState(atItems);
+    const [showThemedModal, setShowThemedModal] = useState(false);
+    const [themedModalType, setThemedModalType] = useState('warning');
+    const [themedModalTitle, setThemedModalTitle] = useState('');
+    const [themedModalContent, setThemedModalContent] = useState(<></>);
+
+    const bulkUpdateReportStatus = async (testPlanReportIds, status) => {
+        try {
+            await triggerLoad(async () => {
+                const result = await bulkUpdateTestPlanReportStatusMutation({
+                    variables: {
+                        testReportIds: testPlanReportIds,
+                        status
+                    }
+                });
+                setTestPlanReports(
+                    result.data.testPlanReport.bulkUpdateStatus.map(
+                        i => i.testPlanReport
+                    )
+                );
+            }, 'Updating Test Plan Status');
+        } catch (e) {
+            showThemedMessage(
+                'Error Updating Test Plan Status',
+                <>{e.message}</>,
+                'warning'
+            );
+        }
+    };
+
+    const showThemedMessage = (title, content, theme) => {
+        setThemedModalTitle(title);
+        setThemedModalContent(content);
+        setThemedModalType(theme);
+        setShowThemedModal(true);
+    };
+
+    const onThemedModalClose = () => {
+        setShowThemedModal(false);
+        dropdownUpdateReportStatusButtonRef.current.focus();
+    };
+
+    let phase = 'Draft'; // also accounts for IN_REVIEW
+    if (testPlanReports.every(i => i.status === 'RECOMMENDED'))
```

Comment:
Should this be `testPlanReports.every(i => i.status === 'RECOMMENDED') || testPlanReports.every(i => i.vendorReviewStatus === 'APPROVED')` ?

Or should the status of the report be moved to `RECOMMENDED` when the vendor approves the report?